client/src/components/v3/DASRecording.tsx

```diff
@@ -8,13 +9,24 @@ import { Button } from 'react-bootstrap';
  * @returns Component
  */
 export default function DASRecording(): JSX.Element {
+
+  // start & stop boosting 
+  function startBoosting(){
+    emit('start-boost');
+    
+  }
+
+  function stopBoosting(){
+    emit('stop-boost');
+  }
+
```

Comment:
This would be better served in `api/common/data` as we want to keep all the external data acquisition (i.e. from socket.io) to the api.

Reply:
easy fix !!

Reply:
remember to re-request review after the requested changes have been addressed!